versions.yaml

```diff
@@ -0,0 +1,33 @@
+---
+# Sources for a base cloud os images
+cloudimg:
+  ubuntu:
+    focal:
+      amd64:
+        url: https://cloud-images.ubuntu.com/releases/focal/release-20230107/ubuntu-20.04-server-cloudimg-amd64.img
+        sha256sum: 3895e38566e5c2c019f5c6f825ab7570ee34dac6b9142fab0c7e5a78084c4280
```

Comment:
Previously, I commented that we should use `sha256sum` instead of `checksum` here to avoid ambiguity, but I'd like to take back my comment. Sorry.

After that, I noticed that Packer's option `iso_checksum` accepts a form of `"sha256:{$checksum}"`.

https://github.com/hashicorp/packer-plugin-sdk/blob/8331daf9a68be731b67b59621179cd3909458986/multistep/commonsteps/iso_config.go#L98-L118

So, I think it is better to specify a hash type in a value like below.

```
url: https://cloud-images.ubuntu.com/releases/focal/release-20230107/ubuntu-20.04-server-cloudimg-amd64.img
checksum: sha256:3895e38566e5c2c019f5c6f825ab7570ee34dac6b9142fab0c7e5a78084c4280
```